README.md

```diff
@@ -1,56 +1,119 @@
-[Racket](https://racket-lang.org/) is a general-purpose programming
-language and an ecosystem for language-oriented programming.
+# Racket Programming Language
 
-This repository holds the source code for the core of Racket plus some
-related packages. The rest of the Racket distribution source code is
-in other repositories, mostly under [the Racket GitHub
-organization](https://github.com/racket).
+![Build](https://img.shields.io/github/actions/workflow/status/racket/racket/ci.yml)
```

Comment:
"ci-push-x86_linux.yml"? I would be inclined to leave out these badges, since the first two, at least, are already displayed by GitHub. But I don't mind if others prefer to keep them.